### PR TITLE
update Author TH

### LINKED ISSUE #9683: Extending PartialTagHelper
Is it possible to extend existing tag helper classes? I am specifically trying to update the TagHelperContext in a class derived from PartialTagHelper to change the name of the view the partial is loading.

Why?
I have minified .cshtml views that I want my partial to load without changing the name property of all my &lt;partial&gt; tag helpers. I have tried to create a new TagHelperContext by overriding the Init function as well as trying to call .Reinitialize in the ProcessAsync method but that doesn't seem to be the extension point I'm looking for

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 19ecfbee-8b84-9ef2-65b6-8c2542f6370f
* Version Independent ID: 9a0cbaa2-65c9-1618-1003-5fd2e7f6192f
* Content: [Author Tag Helpers in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/authoring?view=aspnetcore-2.1#get-started-with-tag-helpers)
* Content Source: [aspnetcore/mvc/views/tag-helpers/authoring.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/mvc/views/tag-helpers/authoring.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #9683
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/authoring?view=aspnetcore-1.0&branch=pr-en-us-12394#load-minified-partial-view-taghelper)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@reZach  please review.

File: aspnetcore/mvc/views/tag-helpers/authoring.md [hunk 1]
```diff
@@ -307,3 +307,12 @@ The tag helpers provide several properties to retrieve content.
 [!code-csharp[](../../views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/TagHelpers/z1AutoLinkerCopy.cs?highlight=5,6,10&range=8-21)]
 
 * Multiple calls to `GetChildContentAsync` returns the same value and doesn't re-execute the `TagHelper` body unless you pass in a false parameter indicating not to use the cached result.
+
+## Load minified partial view TagHelper
+
+The following code:
+
+* Inherits from the [Partial Tag Helper](xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper)
```
Comment (guardrex) [inline aspnetcore/mvc/views/tag-helpers/authoring.md]:
```suggestion
* Inherits from the [Partial Tag Helper](xref:mvc/views/tag-helpers/builtin-th/partial-tag-helper).
```

File: aspnetcore/mvc/views/tag-helpers/authoring.md [hunk 2]
```diff
@@ -307,3 +307,12 @@ The tag helpers provide several properties to retrieve content.
...
+* Loads the minified version when the app is not in the development environment.
```
Comment (guardrex) [inline aspnetcore/mvc/views/tag-helpers/authoring.md]:
```suggestion
* Loads the minified version when the app isn't in the Development environment.
```

Comment (reZach) [inline aspnetcore/mvc/views/tag-helpers/authoring.md]:
```suggestion
* Loads the minified partial view when the app isn't in the Development environment.
```

**Comment by reZach** [comment]
@Rick-Anderson Only one small change I had, everything looks good.

**Comment by reZach** [comment]
@Rick-Anderson Edited my previous comment.

**Comment by Rick-Anderson** [comment]
@reZach  fantastic. I'll add that - or you can.

**Comment by Rick-Anderson** [comment]
@reZach  take a look now.

**Comment by reZach** [comment]
@Rick-Anderson It looks great! Thanks!

**Pull request merged**